test/suites/zombie-tanssi-relay-eth-bridge/test-consensus-bridge.ts

```diff
@@ -0,0 +1,156 @@
+import { beforeAll, describeSuite, expect, afterAll } from "@moonwall/cli";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { spawn, exec } from "node:child_process";
+import { signAndSendAndInclude, waitSessions } from "../../util/block.ts";
+import { ethers } from "ethers";
+
+function execCommand(command: string, options?) {
+    return new Promise((resolve, reject) => {
+        exec(command, options, (error: child.ExecException, stdout: string, stderr: string) => {
+            if (error) {
+                reject(error);
+            } else {
+                resolve({ stdout, stderr });
+            }
+        });
+    });
+}
+
+describeSuite({
+    id: "ZR-01",
+    title: "Zombie Tanssi Relay Test",
+    foundationMethods: "zombie",
+    testCases: function ({ it, context }) {
+        let relayApi: ApiPromise;
+        let ethereumNodeChildProcess;
+        let relayerChildProcess;
+        let alice;
+        let beefyClientDetails;
+
+        beforeAll(async () => {
+            relayApi = context.polkadotJs("Tanssi-relay");
+            const relayNetwork = relayApi.consts.system.version.specName.toString();
+            expect(relayNetwork, "Relay API incorrect").to.contain("dancelight");
+
+            // //BeaconRelay
+            const keyring = new Keyring({ type: "sr25519" });
+            alice = keyring.addFromUri("//Alice", { name: "Alice default" });
+            const beaconRelay = keyring.addFromUri("//BeaconRelay", { name: "Beacon relay default" });
+
+            const txHash = await relayApi.tx.balances
+                .transferAllowDeath(beaconRelay.address, 1_000_000_000_000)
+                .signAndSend(alice);
+            console.log("Transferred money to beacon relay", txHash.toHex());
+
+            ethereumNodeChildProcess = spawn("./scripts/bridge/start-ethereum-node.sh", {
+                shell: true,
+                detached: true,
```

Comment:
Why detached? I believe this only affects whether the child process is killed when the parent process dies. We don't want any processes running after the tests are finished, right?

```suggestion
```

Reply:
The shell script has its own cleanup mechanism that traps some signals and upon receiving the signal, kills all the process in the same process group via `kill 0`. 

I am adding `detached: true` so that process group id is different than the moonwall process. So, that all process except moonwall can be terminated by `kill 0`. If I do not pass it, the `kill 0` in script will terminate moonwall process as well.

Reply:
First time I hear about process group ids, ok let's keep it detached then

Reply:
*resolved*